common/util.go

```diff
@@ -34,3 +34,7 @@ func SuppressDiffWhitespaceChange(k, old, new string, d *schema.ResourceData) bo
 	log.Printf("[DEBUG] Suppressing diff for %v: old=%#v new=%#v", k, old, new)
 	return strings.TrimSpace(old) == strings.TrimSpace(new)
 }
+
+func SupressCaseSensitivity(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
We already have similar function: https://github.com/databricks/terraform-provider-databricks/blob/main/common/resource.go#L427, so we need to unify them...